lib/platform/gitea/index.spec.ts

```diff
@@ -137,9 +137,9 @@ describe('platform/gitea/index', () => {
   ];
 
   const mockComments: ght.Comment[] = [
-    { id: 1, body: 'some-body' },
-    { id: 2, body: 'other-body' },
-    { id: 3, body: '### some-topic\n\nsome-content' },
+    { id: 11, body: 'some-body' },
+    { id: 12, body: 'other-body' },
+    { id: 13, body: '### some-topic\n\nsome-content' },
```

Comment:
Changed for better testing, otherwise we never see the issue and comment id diffenrence.